src/operator/contrib/adaptive_avg_pooling-inl.h

```diff
@@ -43,6 +43,9 @@
 #include "../operator_common.h"
 #include "../mxnet_op.h"
 #include "../mshadow_op.h"
+#if MXNET_USE_MKLDNN == 1
```

Comment:
Please change it to MXNET_USE_ONEDNN.

Reply:
This is a change on v1.x branch so MXNET_USE_MKLDNN is actually correct.

Reply:
I've missed that.